pages/common/r.md

```diff
@@ -0,0 +1,15 @@
+# R
+
+> R language interpreter.
+
+- Call an R interactive shell (REPL):
+
+`R`
+
+- Check R version:
+
+`R --version`
+
+- Run file as input:
```

Comment:
Change this to  - "Execute a file" to keep it consistent with pages of other programming languages.